### PR TITLE
add experimental Beacon API

### PR DESCRIPTION
**Pull request by danielwegener**
Add support for the [Beacon API](https://www.w3.org/TR/2016/WD-beacon-20160204/) (experimental)

### CODE REVIEW

**Comment by danielwegener** [comment]
I guess the travis build will fail - github seems to take a break :)

**Comment by mseddon** [comment]
Github had a nap-time and now travis is broke.  @sjrd can trigger a retest.

File: src/main/scala/org/scalajs/dom/experimental/beacon/package.scala [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     * small HTTP data from the User Agent to a web server.
+     *
+     * MDN
+     *
+     * @param url The url parameter indicates the resolved URL where the data
+     *            is to be transmitted.
+     * @param data The data parameter is the ArrayBufferView, Blob, DOMString,
+     *             or FormData data that is to be transmitted.
+     */
+    def sendBeacon(url: String, data: BodyInit = null): Boolean
```
Comment (sjrd) [inline src/main/scala/org/scalajs/dom/experimental/beacon/package.scala]: This should have `= js.native`. It's not an abstract method.

File: src/main/scala/org/scalajs/dom/experimental/beacon/package.scala [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
```
Comment (sjrd) [inline src/main/scala/org/scalajs/dom/experimental/beacon/package.scala]: This should have `= js.native`. It's not an abstract method.

**Comment by danielwegener** [comment]
@sjrd  :blush: fixed

**Comment by sjrd** [comment]
LGTM

**Pull request merged**